corehq/ex-submodules/casexml/apps/case/management/commands/create_case_fixtures.py

```diff
@@ -10,6 +10,12 @@
 from corehq.apps.data_dictionary.util import add_properties_to_data_dictionary
 from corehq.util.log import with_progress_bar
 
+try:
+    import faker
+except ModuleNotFoundError:
+    print("Error: faker is not installed. Run `pip install -r requirements/dev-requirements.txt`")
+    sys.exit(1)
```

Comment:
I'm not sure we want this at the top level level of the module since it could result in `SystemExit` being raised as an import side-effect. (I'm not sure if there are any code paths in Django that cause a management command to be imported other than by running the command.) Suggestion: put this in a function and call that function to get a reference to `faker`. Something like
```suggestion
def get_faker():
    try:
        import faker
    except ModuleNotFoundError:
        print("Error: faker is not installed. Run `pip install -r requirements/dev-requirements.txt`")
        sys.exit(1)
    return faker
```
Then add this on the first line of the `handle` and `_create_case_structure` methods
```py
        faker = get_faker()
```

Reply:
see c3345a0b1adad3126781ac07489f1d278af82200